### ISSUE TITLE
Book 05: Sentence 056

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#42]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/c5f9b52f-d2d9-4779-8667-635ec8d253e6)

```
# sent_id = Book_05_-_The_Omens.txt:56
# text = No ioan, netetzaujloia in tochin: in jquac aca ichan calaquja, qujtoa in mjllaca, in mjlpan tlaca: ie tlalpoliujz in jchan.
# text[norm] = No ihuan, netetzahuiloya in tochin: in ihcuac aca ichan calaquiya, quihtoa in millaca, in milpan tlacah: ye tlalpolihuiz in ichan.
# text[orig] = No·ioan·,·netetzaujloia·intochin·:·inj·quac·aca·ichan·calaquja·,·qujtoa¶in·mjllaca·,·in·mjlpan·tlaca·:·ietlalpoli¶ujz·injchan·.
# text[osp] = Los aldeanos, y gente rustica: quando vian, que en su casa entraua algun conejo: luego tomauan mal aguero, y concebian en su pecho, que les aujan de robar la casa, [...]
# text[osp_norm] = Los aldeanos, y gente rústica: cuando veían, que en su casa entraba algún conejo: luego tomaban mal agüero, y concebían en su pecho, que les habían de robar la casa, [...]
# text[spa] = También con eso, se tomó como agüero el conejo, cuando entra a la casa de alguien -- así dicen los campesinos, la gente del campo -- entonces su casa se va a asolar.
# references = A&D/171:1
# tagged = 100.00%
# lemmatised = 100.00%
# analysed = 100.00%
# heads = 100.00%
# relations = 96.15%
1	No	no	ADV	_	_	4	advmod	4:advmod	_
2	ioan	huan	NOUN	_	NounType=Relat|Number[psor]=Sing|Person[psor]=3|State=Pos	4	obl	4:obl	_
3	,	,	PUNCT	_	_	4	punct	4:punct	_
4	netetzaujloia	tetzahuia	VERB	_	_	0	root	0:root	_
5	in	in	DET	_	_	6	det	6:det	_
6	tochin	tochin	NOUN	_	State=Abs	4	_	4	_
7	:	:	PUNCT	_	_	12	punct	12:punct	_
8	in	in	SCONJ	_	_	12	mark	12:mark	_
9	jquac	ihcuac	ADV	_	_	12	advmod	12:advmod	_
10	aca	aca	PRON	_	_	11	nmod	11:nmod	_
11	ichan	chantli	NOUN	_	Number[psor]=Sing|Person[psor]=3|State=Pos	12	obl	12:obl	_
12	calaquja	calaqui	VERB	_	Aspect=Imp|Number[subj]=Plur,Sing|Person[subj]=3|Subcat=Intr|Tense=Past|VerbForm=Fin	4	advcl	4:advcl	_
13	,	,	PUNCT	_	_	14	punct	14:punct	_
14	qujtoa	ihtoa	VERB	_	Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Pres|VerbForm=Fin	4	parataxis:sentence	4:parataxis:sentence	_
15	in	in	DET	_	_	16	det	16:det	_
16	mjllaca	millacatl	NOUN	_	Number=Plur	14	nsubj	14:nsubj	_
17	,	,	PUNCT	_	_	20	punct	20:punct	_
18	in	in	DET	_	_	20	det	20:det	_
19	mjlpan	milpan	NOUN	_	NounClass=Loc	20	nmod	20:nmod	_
20	tlaca	tlacatl	NOUN	_	Number=Plur	16	appos	16:appos	_
21	:	:	PUNCT	_	_	23	punct	23:punct	_
22	ie	ye	ADV	_	_	23	advmod	23:advmod	_
23	tlalpoliujz	polihui	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Subcat=Intr|Tense=Fut|VerbForm=Fin	14	parataxis:speech	14:parataxis:speech	_
23.1	tlal	tlalli	NOUN	_	_	_	_	_	_
24	in	in	DET	_	_	25	det	25:det	_
25	jchan	chantli	NOUN	_	Number[psor]=Sing|Person[psor]=3|State=Pos	23	obl	23:obl	_
26	.	.	PUNCT	_	_	4	punct	4:punct	_
```

**Comment by ftyers** [comment]
López Austin has this merged with the following sentence (#40)
> También esto: se agoraba del conejo cuando entraba a la casa de alguno. Decían los campesinos, los hombres de la sementera: "Ya se destruirá su hogar, || o ya alguno [de los suyos] huirá; seguirá el camino del conejo, del ciervo; ya se hará conejo; ya se hará ciervo; ya se convertirá en conejo; ya se convertirá en ciervo."

**Comment by ftyers** [comment]
Questions:
* What is the function of *in tochin* with respect to *netetzahuiloya*?
* Should *tlal* in *tlalpolihuiz* be incorporated, if so with what function, there is a dictionary entry [tlalpolihui](https://nahuatl.wired-humanities.org/content/tlalpolihui)
* Is *in inchan* the subject of *tlalpolihuiz* ?

**Comment by Lguyogiro** [comment]
> What is the function of in tochin with respect to netetzahuiloya?

I think `nsubj`. This is an impersonal passive iirc.

**Comment by Lguyogiro** [comment]
> Should tlal in tlalpolihuiz be incorporated, if so with what function, there is a dictionary entry [tlalpolihui](https://nahuatl.wired-humanities.org/content/tlalpolihui)

If we do analyze the incorporation, I guess it would be `nsubj`, since _polihui_ is intransitive. not sure how we do subject incorporation, with `nsubj:inner`?. Alternatively we could see this as on oblique incorporation maybe ("it disappeared landily haha") but I think that's less likely.

As far as whether to analyze it as incorporation, my feeling is that if it is productive (if we see other nouns incorporated with this function and this verb) and/or if the incorporated meaning is transparent then we should analyze it. For what it is worth in this case, we also see _ixpolihui_ in the codex elsewhere, but that's the only other case of _polihui_ noun incorporation.

**Comment by Lguyogiro** [comment]
> Is in inchan the subject of tlalpolihuiz ?

I don't think so. Molina defines _tlalpolihui_ as "despoblarse y destruirse el pueblo conpestilencia." which makes it hard to imagine the house being the subject. More likely the portent (rabbit) is doing this, in their house. So the `obl` analysis is good for me.

**Comment by ftyers** [comment]
How about:

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/60f5bf80-fa61-4069-8326-2be8945d3d38)

**Comment by Lguyogiro** [comment]
LGTM

**Comment by ftyers** [comment]
Merged in 33cfa15
